src/Main.java

```diff
@@ -1,5 +1,109 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        //1
+        int aFirst = 1000000;
+        byte bFirst = 127;
+        short cFirst = 32767;
+        long dFirst = 2000000L;
+        float eFirst = 5.12f;
+        double fFirst = 3.34234556;
+
+        System.out.println("Значение переменной aFirst с типом int равно " + aFirst);
+        System.out.println("Значение переменной bFirst с типом byte равно " + bFirst);
+        System.out.println("Значение переменной cFirst с типом short равно " + cFirst);
+        System.out.println("Значение переменной dFirst с типом long равно " + dFirst);
+        System.out.println("Значение переменной eFirst с типом float равно " + eFirst);
+        System.out.println("Значение переменной fFirst с типом double равно " + fFirst);
+
+        //2
+        float aSecond = 27.12f;
+        long bSecond = 987678965549L;
+        float cSecond = 2.786f;
+        short dSecond = 569;
+        short eSecond = -159;
+        int fSecond = 27897;
+        byte gSecond = 67;
+
+        //3
+        byte studentsFirstTeacher = 23;
+        byte studentsSecondTeacher = 27;
+        byte studentsThirdTeacher = 30;
+        short totalSheets = 480;
+        int sheetsStudent = totalSheets / (studentsFirstTeacher + studentsSecondTeacher + studentsThirdTeacher);
+        System.out.println("На каждого ученика рассчитано " + sheetsStudent + " листов бумаги");
+
+        //4
+        byte totalBottles = 16;
+        byte time = 2;
+        byte timeFirst = 20;
+        short timePerDay = 24*60;
+        short timePerThreeDay = 3*24*60;
+        int timePerMonth = 30*24*60;
```

Comment:
надо использовать переменные, а не дублировать вычисление
```suggestion
        int timePerMonth = 10*timePerThreeDay;
```